app/src/main/res/values/strings.xml

```diff
@@ -193,6 +193,7 @@
     <string name="main_to">To</string>
     <string name="main_select_currency">Select currency</string>
     <string name="main_send_to">Send to</string>
+    <string name="main_will_be_send_to">Will be sent to</string>
```

Comment:
it's not connected to main, to i would suggest to place it in the block of sell strings